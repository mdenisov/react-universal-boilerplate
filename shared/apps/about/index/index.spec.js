import React from 'react';
import { render } from 'enzyme';

import About from './index';

describe('About', () => {
  test('render correctly', () => {
    const test = render(<About />);
    expect(test).toMatchSnapshot();
  });
});
