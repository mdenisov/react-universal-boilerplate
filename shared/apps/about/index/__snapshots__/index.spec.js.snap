// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About render correctly 1`] = `
<div>
  <h1>
    About
  </h1>
  <div>
    <p>
      We also need to edit our server.jsx file and call Helmet.renderStatic() after ReactDOMServer.renderToString to be able to use the head data in our prerender.
    </p>
    <p>
      We also need to edit our server.jsx file and call Helmet.renderStatic() after ReactDOMServer.renderToString to be able to use the head data in our prerender.
    </p>
    <p>
      We also need to edit our server.jsx file and call Helmet.renderStatic() after ReactDOMServer.renderToString to be able to use the head data in our prerender.
    </p>
  </div>
</div>
`;
